services/usersService.js

```diff
@@ -4,17 +4,26 @@ const usersModel = require("../models/usersModel");
 const post = async (req, res, next) => {
   // TODO validate req body
 
+  //If email is already in db, do not allow them to create account.
+  const response = await usersModel.getUserByEmail(req.body.email)
+  if(response.email){
+    console.log("same email foud")
+    //res.status(400).send({"Error": "Account with this email already exists"})
+  }
+
   try {
     const hashedPassword = await bcrypt.hash(req.body.password, 10);
     const dbRes = usersModel.post({
       email: req.body.email,
       password: hashedPassword,
     });
 
-    console.log(dbRes); // TODO remove
-
+    req.session.email = req.body.email  // NOT SURE IF NEEDED
```

Comment:
I don't think session should be modified since this is for registering. Once a user logs in, I think req.user has all user attributes except password.